main.py

```diff
@@ -10,6 +10,7 @@
 from jinja2 import Template
 import contextlib
 from datetime import datetime
+from pidtree_bcc import utils as u
```

Comment:
`utils` isn't a long word. Prefer just using the `utils` name over `u` to avoid ambiguity.